eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -85,8 +85,13 @@
         }
       },
       "TargetPlatform": {
-        "Win32Api_debug_tests": {
-          "CmakeArgs": " -DBUILD_TESTING=ON -DBUILD_PERFORMANCE_TESTS=ON -DRUN_LONG_UNIT_TESTS=ON -DBUILD_TRANSPORT_CURL=ON -DBUILD_TRANSPORT_WINHTTP=ON -DMSVC_USE_STATIC_CRT=ON",
+        "Win32Api_debug_tests_curl": {
+          "CmakeArgs": " -DBUILD_TESTING=ON -DBUILD_PERFORMANCE_TESTS=ON -DRUN_LONG_UNIT_TESTS=ON -DBUILD_TRANSPORT_CURL=ON -DMSVC_USE_STATIC_CRT=ON",
+          "BuildArgs": "--parallel 8 --config Debug",
+          "PublishMapFiles": "true"
+        },
+        "Win32Api_debug_tests_winhttp": {
```

Comment:
Let's put an explicit `-DBUILD_TRANSPORT_WINHTTP=ON` here?

Reply:
I thought about that, but the default for Windows is to include WINHTTP and I wanted to make sure that was exercised (I don't believe it's used anywhere else).